saleor/core/sitemaps.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.sitemaps import Sitemap
+
+from ..product.models import Product
+
+
+class ProductSitemap(Sitemap):
+
+    def items(self):
+        return Product.objects.only('id', 'name').order_by('-id')
```

Comment:
Are we sure this is sufficient for Saleor? I recall you changing the implementation of `get_absolute_url`?